libs/app-example-states/src/lib/appointment-list-state.service.ts

```diff
@@ -4,45 +4,34 @@ import { Appointment } from '@myorg/app-example-models'
 import { combineLatest, debounceTime, switchMap, tap } from 'rxjs'
 import { AppointmentApiService } from '@myorg/app-example-api-services'
 import { AlertService } from '@myorg/app-example-core'
+import { AppointmentEditFormService } from '@myorg/app-example-forms'
 
-interface AppointmentState {
+interface AppointmentListState {
     appointments: Appointment[]
     selectedAppointment: Appointment | null
     loading: boolean
     search: string
     page: number
+    orderBy: 'asc' | 'desc'
     size: number
     total: number
 }
 
-const initialState: AppointmentState = {
-    appointments: [
-        {
-            vendorUserId: '23rf2',
-            clientUserId: '3232',
-            date: '23232',
-            description: 'hello',
-            endTimestamp: '2323232',
-            notesForClient: 'sfsd',
-            notesForVendor: 'sjkjjjjjj',
-            createdAt: '2323',
-            startTimestamp: '3432232',
-            groupId: '2323',
-            updatedAt: '232332',
-            id: '1',
-        },
-    ],
+const initialState: AppointmentListState = {
+    appointments: [],
     selectedAppointment: null,
     loading: false,
     search: '',
     page: 1,
     size: 10,
     total: 0,
+    orderBy: 'desc'
 }
 
 @Injectable()
-export class AppointmentStateService extends SimpleStore<any> {
+export class AppointmentListStateService extends SimpleStore<AppointmentListState> {
     appointmentApiService = inject(AppointmentApiService)
+    appointmentEditFormService = inject(AppointmentEditFormService)
```

Comment:
We should not inject form service into state service. This may complicate the lifecycle of form and state. Better approach would be to inject both state service and form service inside the component and do the necessary logic there.